pkg/dockerregistry/server/auth.go

```diff
@@ -167,16 +153,57 @@ func (ac *AccessController) Authorized(ctx context.Context, accessRecords ...reg
 					challenge.err = err
 					return nil, challenge
 				}
-
-				return ctx, nil
```

Comment:
Adding the user client to the context is only necessary if any of the code down the line needs to talk to the OpenShift master using the user's supplied token.